$MirOS: ports/plan9/kencc/patches/patch-utils_vl_l_h,v 1.1 2006/03/31 03:50:26 tg Exp $
--- utils/vl/l.h.orig	Thu Mar 30 23:54:45 2006
+++ utils/vl/l.h	Thu Mar 30 23:55:25 2006
@@ -283,7 +283,7 @@ void	dodata(void);
 void	doprof1(void);
 void	doprof2(void);
 long	entryvalue(void);
-void	errorexit(void);
+void	errorexit(void) __attribute__((__noreturn__));
 void	exchange(Prog*);
 int	find1(long, int);
 void	follow(void);
